lib/puppet/face/help/man.erb

```diff
@@ -1,17 +1,17 @@
 <%# encoding: UTF-8%>
-puppet-<%= face.name %>(8) -- <%= face.summary || "Undocumented subcommand." %>
```

Comment:
I'm thinking we should fix whatever caused the `face` method to no longer be in scope, as that could break any other face applications that are using ERB to render output.

Reply:
That was changed because of the ruby -wc warnings, 'face' was set in man.rb, but not used so you get a warning, I changed it to '@face' which fixes the warning, but then gen_manpages fails because it can't find 'face'